src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -38,8 +38,8 @@ $(function () {
     $(".xref").addClass("text-break");
     var texts = $(".text-break");
     texts.each(function () {
-      $(this).text(function (index, text) {
-        return util.breakText(text);
+      $(this).html(function (index, oldHtml) {
+        return util.breakText(oldHtml);
```

Comment:
does it work for html tags?

Reply:
No. The input should not contain html tags, otherwise it may be inserted into a tag and break it.